src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -889,13 +889,13 @@ private void BeginExecuteNonQueryInternalReadStage(TaskCompletionSource<object>
             }
         }
 
-        private void VerifyEndExecuteState(Task completionTask, String endMethod)
+        private void VerifyEndExecuteState(Task asyncResult, String endMethod)
```

Comment:
`asyncResult` seems like just as reasonable a name as `completionTask`, in my opinion. `VerifyEndExecuteState` is only used in two places to verify the asyncResult, which happens to be a Task.